ruby/engine/RubyEngine.hpp

```diff
@@ -25,11 +26,11 @@ class RubyEngine final : public ScriptEngine
   void* getAs_impl(ScriptObject& obj, const std::type_info&) override;
 };
 
+}  // namespace openstudio
+
 extern "C"
 {
-  openstudio::ScriptEngine* makeScriptEngine([[maybe_unused]] const int argc, [[maybe_unused]] const char* argv[]);
+  SCRIPTENGINE_API openstudio::ScriptEngine* makeScriptEngine(int argc, char* argv[]);
```

Comment:
declspec(dllexport) + wrong location of export symbol, should be outside the openstudio namespace